### PR TITLE
Javascript javascript3 week2

### PR DESCRIPTION
**Pull request by dan-nil**
Homework Javascript 3 - Week 2

### CODE REVIEW

File: javascript/javascript3/week2/currency-calculator-app/script.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    const data = await response.json();
+    const exchangeRate = await data.rates[currencyTo];
+    const totalAmount = amount.value;
+
+    return (totalAmount * exchangeRate).toFixed(2);
+}
+
+// Displaying the currency selections
+async function loadCurrencies(currency = 'EUR') {
+    try {
```
Comment (LucasIversen) [inline javascript/javascript3/week2/currency-calculator-app/script.js:31]: Think about what you put in your try. I'd only put code that might raise or throw an exception

File: javascript/javascript3/week2/currency-calculator-app/script.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
+const amount = document.querySelector('#amount');
+const fromCurrency = document.querySelector('#from-currency');
+const toCurrency = document.querySelector('#to-currency');
+const button = document.querySelector('#convert');
+const result = document.querySelector('#result');
+
+loadCurrencies();
```
Comment (LucasIversen) [inline javascript/javascript3/week2/currency-calculator-app/script.js]: It's good syntax to put the function call underneath your function definition, as it improves readability

File: javascript/javascript3/week2/currency-calculator-app/script.js [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+            toCurrency.appendChild(option2);
+        });
+
+        toCurrency.value = 'DKK';
+    } catch (err) {
+        console.log(err);
+    }
+}
+
+fromCurrency.addEventListener('change', async () => {
```
Comment (LucasIversen) [inline javascript/javascript3/week2/currency-calculator-app/script.js]: This does not need to be async

File: javascript/javascript3/week2/currency-calculator-app/README.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+:white_check_mark: User can choose a currency to convert from(default should be EUR).
+
+:white_check_mark: User can choose a currency to convert to(Default should be DKK).
+
+:white_check_mark: Whenever amount, currency from or currency to changes we show what the amount translates to in the to currency.
+
+#
+
+### Examples
```
Comment (reneHYF) [inline javascript/javascript3/week2/currency-calculator-app/README.md:29]: Test test

**Review by LucasIversen — COMMENTED** [review]
Hey hey. Hopefully you can see this.. Good job on your homework :)

File: javascript/javascript3/week2/currency-calculator-app/script.js (hunk 2 above)
Comment (dan-nil) [reply javascript/javascript3/week2/currency-calculator-app/script.js]: I moved the function call under the function definition.

File: javascript/javascript3/week2/currency-calculator-app/script.js (hunk 3 above)
Comment (dan-nil) [reply javascript/javascript3/week2/currency-calculator-app/script.js]: Removed the async

File: javascript/javascript3/week2/currency-calculator-app/script.js (hunk 1 above)
Comment (dan-nil) [reply javascript/javascript3/week2/currency-calculator-app/script.js:31]: I didn't think about that. 
I fixed the code.

**Comment by dan-nil** [comment]
Thank you for taking the time.
I had a lot to take from your comments.

**Pull request merged**